src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -43,13 +43,17 @@ private void readObject(java.io.ObjectInputStream in)
             doc="Read filters to be disabled before analysis", optional=true, common = true)
     public final List<String> disableFilters = new ArrayList<>();
 
-    @Argument(fullName = "disableAllReadFilters",
-            shortName = "disableAllReadFilters",
-            doc = "Disable all read filters", common = true, optional = true)
-    public boolean disableAllReadFilters = false;
+    @Argument(fullName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_READ_FILTERS,
+            shortName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_READ_FILTERS,
+            doc = "Disable all tool default read filters", common = true, optional = true)
+    public boolean disableToolDefaultReadFilters = false;
 
     // Map of read filter (simple) class names to the corresponding discovered plugin instance
     private Map<String, ReadFilter> readFilters = new HashMap<>();
+    // readFilters map is overridden in validateArguments
+    // this set is for keeping the names for the help in alphabetical order
+    // and to use them for checking if an argument exists or not
+    private final Set<String> readFilterNames = new TreeSet<>();
```

Comment:
I meant the field should be removed altogether. readFilters is overwritten as you point out, but it doesn't need to be. So I've removed it in my set of changes.